website/src/pages/index.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import clsx from 'clsx';
+import Layout from '@theme/Layout';
+import Link from '@docusaurus/Link';
+import useDocusaurusContext from '@docusaurus/useDocusaurusContext';
+import styles from './index.module.css';
+import HomepageFeatures from '../components/HomepageFeatures';
+
+function HomepageHeader() {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <header className={clsx('hero hero--primary', styles.heroBanner)}>
+      <div className="container">
+        <h1 className="hero__title">{siteConfig.title}</h1>
+        <p className="hero__subtitle">{siteConfig.tagline}</p>
+        <div className={styles.buttons}>
+          <Link
+            className="button button--secondary button--lg"
+            to="/docs/introduction/getting-started">
+            Getting Started with Detox
+          </Link>
+            <iframe className={styles.githubStar}
+                    src='https://ghbtns.com/github-btn.html?user=wix&repo=detox&type=star&count=true&size=large'
+                    frameBorder='0' scrolling='0' width='170' height='30' title='GitHub'/>
+        </div>
+      </div>
+    </header>
+  );
+}
+
+export default function Home() {
+  const {siteConfig} = useDocusaurusContext();
+  return (
+    <Layout
+      title={`Hello from ${siteConfig.title}`}
```

Comment:
Is this not finished?

Reply:
this text is invisible, anyway - removed.